test/persistence-hooks.suite.js

```diff
@@ -736,10 +739,18 @@ module.exports = function(dataSource, should) {
           { id: existingInstance.id, name: 'updated name' },
           function(err, instance) {
             if (err) return done(err);
-            observedContexts.should.eql(aTestModelCtx({
-              where: { id: existingInstance.id },
-              data: { id: existingInstance.id, name: 'updated name' }
-            }));
+            if (dataSource.connector.findOrCreate) {
+              observedContexts.should.eql(aTestModelCtx({
+                where: { id: existingInstance.id },
+                data: { id: existingInstance.id, name: 'updated name' }
```

Comment:
Typo, you are executing `updateOrCreate` but testing for `findOrCreate`.

Atomic findOrCreate does not set `ctx.currentInstance` - we need to capture this in the documentation.